rpg.py

```diff
@@ -138,17 +145,21 @@ def run(self):
                 self.players = updated_players
                 for player in self.players:
                     if player.is_alive():
-                        print(f'{player.name}')
+                        print(f'{player.name}: {player.level}')
                         print(f'HP: {player.hp}')
                 if acting_player.is_alive and acting_player.get_all_enemies(self.players):
                     self.add_into_queue(acting_player, current_game_time)
             else:
                 print(f'{acting_player.name} is dead')
 
         print('The battle is over')
-        print('The following players survived:')
-        print(f'{[player.name for player in self.players if player.is_alive()]}')
-        
+    for player in self.players:
+        if player.is_alive():
+            if player.team == 1:
+                print('You win!')
```

Comment:
This repeats the code for every player that is alive on the certain team. I do not want this, I only want to define the winner once.